pgjdbc/src/test/java/org/postgresql/test/jdbc2/ArrayTest.java

```diff
@@ -336,6 +338,99 @@ public void testSetPrimitiveArraysObjects() throws SQLException {
     rs.close();
   }
 
+  @Test
+  public void testSetArraysWithAnsiTypeNames() throws SQLException {
+    TestUtil.createTable(
```

Comment:
this should be in setUp() so that if there are errors the table will be dropped.

Reply:
Hm... that's a good point, but it might be a bit much to create and drop the table for every test method when it's only used by this one. Since it's specific to this test, maybe it makes more sense to wrap this test in a try / finally? (If not, I can certainly move this to `setUp()` instead.)

Reply:
That works for me. As long as it's cleaned up in the finally.

Reply:
Done :+1: